### ISSUE TITLE
WARNING - Did not clean all fastq/fasta files

### ISSUE THREAD

**Issue by bbessesen** [faircloth-lab/phyluce#333]
I'm having a bit of trouble with my phyluce assembly. Hping you can advise me. Illumiprocessor ran fine, and the cleaned_read contents and assembly.conf all seem to look good. But when I ran the assembly code (using spades), the process went quite fast, <8 minutes per assembly, and each gave the same warning (see example text below). I’m running everything on an institutional cluster with 20 cores and 300 RAM, though I did not designate any amount of memory in the assembly code, which I’m now reading could be important—would doing that solve this issue?

2024-03-27 09:06:59,416 - phyluce_assembly_assemblo_spades - INFO - Symlinking assembled contigs into /home/FM/bbessesen/sqcl/spades-assemblies/contigs
2024-03-27 09:06:59,416 - phyluce_assembly_assemblo_spades - INFO - -------- Processing Hydrophis_p_platurus_BB12_FMNH171682 --------
2024-03-27 09:06:59,416 - phyluce_assembly_assemblo_spades - INFO - Finding fastq/fasta files
2024-03-27 09:06:59,425 - phyluce_assembly_assemblo_spades - INFO - File type is fastq
2024-03-27 09:06:59,425 - phyluce_assembly_assemblo_spades - INFO - Running SPAdes for PE data
2024-03-27 09:07:28,651 - phyluce_assembly_assemblo_spades - WARNING - Did not clean all fastq/fasta files from /home/FM/bbessesen/sqcl/spades-assemblies/Hydrophis_p_platurus_BB12_FMNH171682_spades

**Comment by brantfaircloth** [comment]
Yes, increase the RAM available to spades using the flag for the code (`--memory`), as well as using a compute node w/ a fair bit of RAM.

**Comment by bbessesen** [comment]
Ok, thanks for the quick reply, I'll give it a try.

**Comment by bbessesen** [comment]
I'm calling the run as: $ phyluce_assembly_assemblo_spades --config assembly.conf --output spade-assemblies --memory 200 --cores 20
It's giving me an error: unrecognized arguments: --memory 200

**Comment by brantfaircloth** [comment]
Make sure you are using an up-to-date version and also alter your output directory name to remove the hyphen.

**Comment by bbessesen** [comment]
I changed my output directory name, and our IT guy kindly updated the system to 1.7.3, and voila, it seems to be running. :) Many thanks for the great help!!
